src/test/kotlin/graphics/scenery/tests/examples/compute/PersistentTextureRequestsExample.kt

```diff
@@ -0,0 +1,143 @@
+                SegmentType.FragmentShader to SegmentTemplate(
+                    this.javaClass,
+                    "ComputeVolume.comp",
+                    "intersectBoundingBox", "vis", "SampleVolume", "Convert", "Accumulate"),
+            ))
+
+        val outputBuffer = MemoryUtil.memCalloc(1280*720*4)
+        val outputTexture = Texture.fromImage(Image(outputBuffer, 1280, 720), usage = hashSetOf(Texture.UsageType.LoadStoreImage, Texture.UsageType.Texture))
+        volumeManager.material.textures["OutputRender"] = outputTexture
+
+        hub.add(volumeManager)
+
+        val imp: ImagePlus = IJ.openImage("https://imagej.nih.gov/ij/images/t1-head.zip")
+        val img: Img<UnsignedShortType> = ImageJFunctions.wrapShort(imp)
+
+        val volume = Volume.fromRAI(img, UnsignedShortType(), AxisOrder.DEFAULT, "T1 head", hub, VolumeViewerOptions())
+        volume.transferFunction = TransferFunction.ramp(0.001f, 0.5f, 0.3f)
+        scene.addChild(volume)
+
+        val box = Box(Vector3f(1.0f, 1.0f, 1.0f))
+        box.name = "le box du win"
+        box.material.textures["diffuse"] = outputTexture
+        box.material.metallic = 0.0f
+        box.material.roughness = 1.0f
+
+        scene.addChild(box)
+
+        val light = PointLight(radius = 15.0f)
+        light.position = Vector3f(0.0f, 0.0f, 2.0f)
+        light.intensity = 5.0f
+        light.emissionColor = Vector3f(1.0f, 1.0f, 1.0f)
+        scene.addChild(light)
+
+        val cam: Camera = DetachedHeadCamera()
+        with(cam) {
+            position = Vector3f(0.0f, 0.0f, 5.0f)
+            perspectiveCamera(50.0f, 512, 512)
+
+            scene.addChild(this)
+        }
+
+        thread {
+            val opTexture = volumeManager.material.textures["OutputRender"]!!
+
+            var prevCounter = 0
+
+            (renderer as? VulkanRenderer)?.persistentTextureRequests?.add(opTexture to counter)
+
+            // this is the loop where you may do your tasks that are asynchronous to rendering
+            while (renderer?.shouldClose == false) {
+                while (counter.get() == prevCounter) {
+                    Thread.sleep(5)
+                }
+                prevCounter = counter.get()
+
+                //updated texture has been returned
+                val buffer = opTexture.contents
+                //the buffer can now, e.g., be transmitted, as is required for parallel rendering
+
+                if (buffer != null && prevCounter == 100) {
+                    SystemHelpers.dumpToFile(buffer, "texture-${SystemHelpers.formatDateTime(delimiter = "_")}.raw")
+                }
+            }
+        }
+
+        thread {
+            while (renderer?.firstImageReady == false) {
+                Thread.sleep(5)
+            }
+
+            Thread.sleep(1000) //give some time for the rendering to take place
+
+            renderer?.close()
+            Thread.sleep(200) //give some time for the renderer to close
```

Comment:
@skalarproduktraum is there a way to query whether the renderer has terminated? Here, if the renderer does not close within 200 ms, the value of totalFrames recorded is incorrect. This leads to the test sometimes failing on CI, depending on whether the renderer closed in 200 ms or not